github/users.go

```diff
@@ -144,6 +144,22 @@ type UserListOptions struct {
 
 // ListAll lists all GitHub users.
 //
+// To paginate through all users, populate 'Since' with the ID of the last user.
+// Example (warning: this will quickly eat through your GitHub API quota!):
```

Comment:
In general, I really like real Go examples and think that it improves the documentation.
However, I'm concerned about making this particular one a real example because then it will actually be run during "go test" which we definitely do not want to do.
So I'll leave this as-is. Thanks!